README.md

```diff
@@ -56,7 +56,8 @@ jobs:
 | FONT_SIZE      | Font size of name (px)                                      | no       | `14`                                          |
 | PATH           | Path to the file you want to add contributors' list         | no       | `/README.md`                                  |
 | COMMIT_MESSAGE | commit message                                              | no       | `docs(README): update contributors`           |
-| AVATAR_SHAPE   | Set `round` for round avatar and `square` for square avatar | no       | square                                        |
+| AVATAR_SHAPE   | Set `round` for round avatar and `square` for square avatar | no       | square   
```

Comment:
Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/manual/bobankhshen/add-contributors/pullRequest?prid=6018022)